deploy/scripts/src/mobile/testSteps/backend.ts

```diff
@@ -54,10 +57,16 @@ export function getBiometricTokenV2 (sessionId: string): void {
     const biometricTokenUrl = buildBackendUrl('/biometricToken/v2', {
       authSessionId: sessionId
     })
+
+    const startTime = Date.now()
```

Comment:
Why do we need it for this request? This is a BE API that returns a 2XX for a happy scenario (no redirect).

@Tom-Dann - do we need to be consistent with this timing approach, i.e. apply it to every single request, or can we only apply it where we know we have redirects, and want to track the overall request time.

Reply:
It makes sense to do it for every request, then you have one metric that can capture the total duration regardless of if it has redirects or not

Reply:
They are all sent by default, so this one in particular will be accessible as `k6.duration` 
![image](https://github.com/alphagov/di-devplatform-performance/assets/110121463/be8c1fea-a669-46b8-b580-710713d6efc6)